diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 2a60ac262584a6cdae0dbdc3e73058be914c0f6c..3dd4e3d59cbe89eeeb4d3499cd422a64e317190d 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -44,6 +44,7 @@ exports.networks = [
     { chainId: 109n, shortName: 'shibariumecosystem' },
     { chainId: 111n, shortName: 'etl' },
     { chainId: 114n, shortName: 'c2flr' },
+    { chainId: 1868n, shortName: 'soneium' },
     { chainId: 122n, shortName: 'fuse' },
     { chainId: 123n, shortName: 'spark' },
     { chainId: 133n, shortName: 'HSKT' },
